LEGO1/mxpalette.h

```diff
@@ -18,7 +18,7 @@ class MxPalette : public MxCore
   virtual ~MxPalette();
 
   void ApplySystemEntriesToPalette(LPPALETTEENTRY p_entries);
-  MxPalette* Clone();
+  MxPalette& Clone();
```

Comment:
Incorrect. `Clone` (in any context) always returns a _pointer_ to a new object.

Reply:
fixed locally